pkg/iphelpers/iphelpers_test.go

```diff
@@ -571,6 +572,190 @@ var _ = Describe("DecIPAddress operations", func() {
 	})
 })
 
+var _ = Describe("GetIPRange operations", func() {
+	It("creates an IPv4 range properly for 30 bits network address", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.21.100/30")
+		ip, _ := AddressRange(ipnet)
+		Expect(err).NotTo(HaveOccurred())
+
+		firstip, lastip, err := GetIPRange(net.ParseIP(ip.String()), *ipnet)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(fmt.Sprint(firstip)).To(Equal("192.168.21.101"))
+		Expect(fmt.Sprint(lastip)).To(Equal("192.168.21.102"))
+
+	})
+	It("creates an IPv4 range properly for 24 bits network address with different range start", func() {
+
+		_, ipnet, err := net.ParseCIDR("192.168.2.200/24")
+		ip := net.ParseIP("192.168.2.23") // range start
+
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
ditto: please move this immediately below the call that throws it.

Reply:
same comment as above